appeal/service.go

```diff
@@ -174,11 +174,17 @@ func (s *Service) Create(appeals []*domain.Appeal) error {
 		}
 		a.Policy = policies[policyConfig.ID][uint(policyConfig.Version)]
 
+		creatorDetails, err := s.iamService.GetUser(a.CreatedBy)
+		if err != nil {
+			return fmt.Errorf("fetching creator's user iam: %w", err)
+		}
+		a.Creator = creatorDetails
+
 		approvals := []*domain.Approval{}
 		for i, step := range a.Policy.Steps { // TODO: move this logic to approvalService
-			var approvers []string
-			if step.Approvers != "" {
-				approvers, err = s.resolveApprovers(a.AccountID, a.Resource, step.Approvers)
+			var approverEmails []string
+			if step.Strategy == "manual" {
```

Comment:
should we change hardcoded "manual" to enum?

Reply:
updated @StewartJingga